api/agent_test.go

```diff
@@ -154,3 +156,87 @@ func TestAgent_SetServers(t *testing.T) {
 		t.Fatalf("bad server list: %v", out)
 	}
 }
+
+func (a *AgentMember) String() string {
+	return "{Name: " + a.Name + " Region: " + a.Tags["region"] + " DC: " + a.Tags["dc"] + "}"
+}
+
+func TestAgents_Sort(t *testing.T) {
+	var sortTests = []struct {
+		in  []*AgentMember
+		out []*AgentMember
+	}{
+		{
+			[]*AgentMember{
+				&AgentMember{Name: "nomad-2.vac.us-east", Tags: map[string]string{"region": "us-east", "dc": "us-east-1c"}},
```

Comment:
Can you make the structs span multiple lines. We try to keep with to ~80 characters per line